frontend/components/service/durationPrice.js

```diff
@@ -1,66 +1,66 @@
 import Card from '@mui/material/Card';
-import { InputAdornment, Slider } from '@mui/material';
+import {InputAdornment, Slider} from '@mui/material';
```

Comment:
shouldn't a white space being added automatically inside the { }?

Reply:
yes, there is a setup of prettier to run all the code when creating a commit, so line 2 pink changes to line 2 blue. Remove the white space.